change/@ni-nimble-components-df093e6a-7cba-46ed-8d5d-07670af90719.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I have a personal note to expose the `lang` property in the Angular directive for the theme provider, and that will include the usual tests. Why do we need to additionally test changing the `lang` in SLE? We do have a test for the date-text column that changing the `lang` token affects the rendered string.

Reply:
My understanding of @jattasNI's statement is that we need to verify (at least manually) that changing the locale in SLE sets the appropriate language on the page such that the `nimble-theme-provider` picks up that value.

Looking at what happens today, SLE does set the `lang` attribute on `document.documentElement`, but it is just the two letter language code (e.g. `de` rather than `de-DE`). Will that work as expected, or do we need to make an SLE change to have it set the longer language code?

Reply:
It may be dependent on what is using it. For example, the `Intl.DateTimeFormat` constructor supports a `localeMatcher` option that affects how the runtime finds a suitable locale to use based on what was requested. I'm willing to bet that most locale-aware code would work similarly though, such that the region subtag isn't necessary to resolve to a suitable locale.